### PR TITLE
Fix ordering of kubectl apply arguments

### PR DESCRIPTION
**Pull request by ciaranevans**
## Summary

TL;DR: The `-f` argument in `kubectl apply` has to be beside the `-` argument, without it you get an error that looks like:

```
$ prefect agent kubernetes install -t "<TOKEN>" --rbac -n "<namespace>" | kubectl create -f --namespace="<namespace>" -
error: Unexpected args: [-]
See 'kubectl create -h' for help and examples
```

## Changes

Updates `docs/orchestration/agents/kubernetes.md` to show the correct ordering of arguments

## Importance

Currently the Prefect provided instructions on applying the Prefect Agent configuration to Kubernetes clusters is broken.

## Checklist

This PR:

- [X] adds new tests (if appropriate) N/A
- [X] adds a change file in the `changes/` directory (if appropriate) N/A
- [X] updates docstrings for any new functions or function arguments, including `docs/outline.toml` for API reference docs (if appropriate) N/A

### CODE REVIEW

**Review by zangell44 — APPROVED** [review]
Looks good to me. Thanks for taking the time to update the docs @ciaranevans!

**Comment by ciaranevans** [comment]
Thanks @zangell44 !
